### PR TITLE
layers: Improve ValidateCmdQueueFlags message

### PR DESCRIPTION
**Pull request by sfricke-samsung**

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -1869,10 +1869,12 @@ bool CoreChecks::ValidateCmdQueueFlags(const CMD_BUFFER_STATE *cb_node, const ch
                                        const char *error_code) const {
     auto pool = cb_node->command_pool.get();
     if (pool) {
-        VkQueueFlags queue_flags = GetPhysicalDeviceState()->queue_family_properties[pool->queueFamilyIndex].queueFlags;
+        const uint32_t queue_family_index = pool->queueFamilyIndex;
+        const VkQueueFlags queue_flags = GetPhysicalDeviceState()->queue_family_properties[queue_family_index].queueFlags;
         if (!(required_flags & queue_flags)) {
             string required_flags_string;
-            for (auto flag : {VK_QUEUE_TRANSFER_BIT, VK_QUEUE_GRAPHICS_BIT, VK_QUEUE_COMPUTE_BIT}) {
+            for (auto flag : {VK_QUEUE_TRANSFER_BIT, VK_QUEUE_GRAPHICS_BIT, VK_QUEUE_COMPUTE_BIT, VK_QUEUE_SPARSE_BINDING_BIT,
```
Comment (ncesario-lunarg) [inline layers/core_validation.cpp:1876]: Looks like this could be `const` (I realize it wasn't like that before)?

**Review by ncesario-lunarg — APPROVED** [review]
@mark-lunarg This change is passing internal CI

File: layers/core_validation.cpp (hunk 1 above)
Comment (sfricke-samsung) [reply layers/core_validation.cpp:1876]: sorry was slow on this,  did you want to make a follow PR on this for me @ncesario-lunarg , thanks!

Comment (ncesario-lunarg) [reply layers/core_validation.cpp:1876]: No worries, I don't think so. Looks like there are lots of places like this (i.e., that _probably_ should be `const`), and I just happen to notice this one due to the review; maybe something we can change in the future :) Thanks!

**Pull request merged**